sdk/confidentialledger/Azure.Security.ConfidentialLedger/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/confidentialledger/Azure.Security.ConfidentialLedger",
+  "Tag": "net/confidentialledger/Azure.Security.ConfidentialLedger_081caadb5d"
```

Comment:
Where does `081caadb5d` come from? Would it typically be auto-generated?

Reply:
It is actually the SHA of assets repo when creating the tag :)

So you have a tag that you point to, but that tag also points at the actual data reference itself.

Reply:
I should be clear, I am intending on backing out my `assets.json` change. I will not be moving _anyone_ that doesn't explicitly do themselves. This is messing with your day-to-day dev and I totally realize that👍

Reply:
No worries - my comments are purely about the approach.